test/pretty-print.test.js

```diff
@@ -354,3 +354,22 @@ test('pretty print - includeMeta, includeHooks', t => {
     t.equal(hooksOnly, hooksOnlyExpected)
   })
 })
+
+test('pretty print - mixed parametic routes with HEAD method generated by fastify', t => {
+  t.plan(2)
+  const fastify = Fastify()
+
+  fastify.get('/:other', () => {})
+  fastify.get('/example', () => {})
+
+  fastify.ready(() => {
+    const routes = fastify.printRoutes()
+    const expectedRoutes = `\
+└── /
+    ├── example (GET, HEAD)
+    └── :other (GET, HEAD)
+`
+    t.equal(typeof routes, 'string')
+    t.equal(routes, expectedRoutes)
```

Comment:
This should be enough, if `routes` is undefined, the same `t.equal` will catch it

```suggestion
    t.equal(routes, expectedRoutes)
```